tests/e2e/specs/menu-top/LoadModal.spec.js

```diff
@@ -0,0 +1,50 @@
+describe('components/menu-top/LoadModal', () => {
+  beforeEach(() => {
+    cy.visit('/');
+
+    cy.get('[data-test="menu-top--file"]')
+      .click();
+
+    cy.get('[data-test="menu-top--load-show"]')
+      .click();
+
+    cy.get('[data-test="menu-top--load-modal"]')
+      .should('be.visible');
+  });
+
+  it('renders with the defaults', () => {
+    cy.get('[class="modal-card-title"]')
```

Comment:
Let's add a `data-test` attribute for the modal-card-title so we aren't referencing the class
https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements

Reply:
Good call.  Done